.github/workflows/publish.yml

```diff
@@ -0,0 +1,28 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Publish
+
+# Controls when the action will run. Triggers the workflow on push or pull request 
+# events but only for the master branch
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    if: github.ref == 'refs/heads/master'
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: true
+      - name: Build Docker image
+        run: make build
```

Comment:
https://github.com/libero/reviewer-reporter/blob/master/docker-compose.yaml#L7 should be `libero/...`; the `liberoadmin/...` rename is done by the push script and was always supposed to be temporary

Reply:
Changed :+1:

Reply:
Changed and raised https://github.com/libero/reviewer/issues/552